app/scripts/components/common/page-header/nav/nav-item-cta.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import GoogleForm from '../../google-form';
+import { useFeedbackModal } from '../../layout-root';
+import { ActionNavItem } from '../types';
+
+interface NavItemCTAProps {
+  item: ActionNavItem;
+}
+
+export const NavItemCTA: React.FC<NavItemCTAProps> = ({
+  item
+}): JSX.Element => {
+  const { isRevealed, show, hide } = useFeedbackModal();
+  return (
+    <React.Fragment key={item.id}>
+      {item.actionId === 'open-google-form' && (
+        <>
+          <a className='usa-nav__link' id={item.id}>
```

Comment:
without href, a tag doesn't get keyboard focus (And its hover style doesn't show the cursor.) Can we give some dummy href to this a tag, and move the onclick event to `a` tag?